admin_app/src/components/NavBar.tsx

```diff
@@ -28,12 +28,13 @@ const settings = ["Logout"];
 const NavBar = () => {
   return (
     <AppBar
-      position="static"
+      position="fixed"
```

Comment:
I needed to make this change for the Drawer used for the Main Menu to not cover the navbar. This also meant I had to change the `layout.tsx` for other pages.

Reply:
Oh okay thanks, answered my question on `urgency-rules/layout.tsx`